src/utilcode/util.cpp

```diff
@@ -1219,16 +1219,32 @@ int GetCurrentProcessCpuCount()
             count = 64;
     }
 
+    cCPUs = count;
+
+    return count;
+}
+
+int GetCurrentProcessCpuLimit()
```

Comment:
I have realized that this function now actually returns the number of CPUs with applied limit if any. It seems that if you renamed this function to GetCurrentProcessCpuCountLimited or something along these lines, its caller sites can be simplified. They could just call this function instead of calling the GetCurrentProcessCpuCount, this function and having the ASSERT and  "if".